plugins/MakerbotWriter/MakerbotWriter.py

```diff
@@ -143,7 +161,7 @@ def _getMeta(self, root_nodes: List[SceneNode]) -> Dict[str, any]:
                         nodes.append(node)
 
         meta = dict()
-
+        file_format = global_stack.definition.getMetaDataEntry("file_formats")
```

Comment:
I get why you did it like this. I had a different solution in mind where we would basically duplicate the whole plugin and each plugin would listen to a different mimetype. This solution is clearly "easier" to implement/understand but it does go a bit against the intended use of plugins. @nallath do you maybe have an idea how to implement this in a better way?

Reply:
I believe the other way would be to create a separate plugin for sketch printers. Or if we can somehow use limit mimetypes from the printer definitions? and then have this information to use in plugin?

Reply:
Duplicating is perhaps a bit word? I'd say that it's mostly inheritance with a specialization that overrides the mimetype.

Reply:
https://github.com/Ultimaker/Cura/pull/19029/commits/21b8694289a89bb5319a9b3ebc0246d80b88076a